### DISCUSSION TITLE
Mysql course using dbeaver

### DISCUSSION THREAD

**Discussion by josnailai94** [dbeaver/dbeaver#10757] [General]
I am new to Mysql and SQL in general. Im looking to take a course to learn it but all the udemy and other courses I can find use workbench and/or just code. Any suggestions of a good course online which teaches mySQL using DBeaver? Even a tutorial on creating a basic database w Dbeaver would probably suffice.

**Comment by serge-rider** [comment]
Well, we have a sort of tutorial in our wiki: https://github.com/dbeaver/dbeaver/wiki

But not any real courses. Yet, I hope.
